lib/fetcher.js

```diff
@@ -15,6 +15,7 @@ const fsm = require('fs-minipass')
 const cacache = require('cacache')
 const osenv = require('osenv')
 const isPackageBin = require('./util/is-package-bin.js')
+const getContents = require('@npmcli/installed-package-contents')
```

Comment:
nit: just making sure that this package has passing tests

Reply:
It does. https://github.com/npm/installed-package-contents/runs/351565160